service/matching/taskListManager.go

```diff
@@ -166,7 +165,15 @@ func newTaskListManager(
 			metrics.MatchingTaskListMgrScope,
 		))
 	}
-
+	var taskListTypeTag metrics.Tag
+	if taskList.taskType == persistence.TaskListTypeActivity {
+		taskListTypeTag = metrics.TaskListTypeTag("activity")
+	} else {
+		taskListTypeTag = metrics.TaskListTypeTag("decision")
```

Comment:
nit: define global variables for those two tags somewhere in the matching package?